_posts/2017-08-25-aug-25.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: "This Week In Erlang Aug 25"
+description: ""
+date: 2017-08-25
+tags: [erlang, news]
+comments: false
+share: true
+---
+
+Hello and welcome to "This week in Erlang"! As expected, there was a lot of action on the OTP side this week, and maybe a little too slow everywhere else. However, there was a piece of great news that came out this week. Read on!
+
+### Articles and Blog posts
+ * Bet365's CEO announced that they will be buying all of Basho's remaining assets and making everything open source: <http://lists.basho.com/pipermail/riak-users_lists.basho.com/2017-August/019500.html>
+
+ * There was a discussion on Hackernews about Pierre Fenoll's Supercompiler Pass project: <https://news.ycombinator.com/item?id=15064127>
+
+ * Andrew Bennett (@potatosaladx) wrote another great article this week. It is about load testing Cowboy 2.0 and how it holds against Cowboy 1.1. Highly recommend reading it: <https://potatosalad.io/2017/08/20/load-testing-cowboy-2-0-0-rc-1.html>
```

Comment:
I found 3 misspelled words in this sentence: 
- <https
- Bennett
	- *Suggested Replacements:* sennett, bennets, bennet
- *
	- *Suggested Replacements:* , k, p, u, e, z, j, o, t, d, y, i, n, s, c, x, h, m, r, b, w, g, l, q, a, v, f